### ISSUE TITLE
ADA - links

### ISSUE THREAD

**Issue by robynlloos** [uno-isqa-8950/uno-cpi#1416]
All linked words should be underlined with a thin line.

For example:
![image](https://user-images.githubusercontent.com/44984230/63383883-44811680-c363-11e9-9281-e9076fe566e7.png)
